repos/fdbt-site/src/pages/viewPassengerTypes.tsx

```diff
@@ -185,7 +185,7 @@ const PassengerTypeGroups = ({ passengerTypeGroups }: { passengerTypeGroups: Gro
                                     {passengerTypeGroup.maxGroupSize}
                                 </p>
 
-                                {passengerTypeGroup.companions.length
+                                {passengerTypeGroup?.companions?.length
                                     ? passengerTypeGroup.companions.map((companion) => (
```

Comment:
Do you need the length check now or can you just do `passengerTypeGroup?.companions?.map`

Reply:
I think we do. I just did some testing in the Chrome console. An empty array still throws an exception if you try to do `.map()` on it.